src/FormDocument.js

```diff
@@ -0,0 +1,123 @@
+                {errors.achternaam && <p>{errors.achternaam.message}</p>}
+            </label>
+            <label htmlFor="age">Leeftijd:
+                <input
+                    type="number"
+                    id="age"
+                    name="age"
+                    placeholder="are you 18..?"
+                    {...register("age", {
+                        min: {
+                            value: 18,
+                            message: "je bent geen 18..."
+                        }
+                    })}
+                />
+                {errors.age && <p>{errors.age.message}</p>}
+            </label>
+            <label htmlFor="postal-code">Postcode:
+                <input
+                    type="text"
+                    id="postal-code"
+                    name="postcode"
+                    {...register("postcode", {
+                        required: { value: true, message: "Dit veld is verplicht" },
+                        pattern: {
+                            value: /^[1-9][0-9]{3}?(?!sa|sd|ss)[a-z]{2}$/im,
+                            message: "niet geldige postcode",
+                        }})}
+                />
+                {errors.postcode && <p>{errors.postcode.message}</p>}
+            </label>
+            </div>
+
+            <br />
+
+            <label htmlFor="Bezorgfrequentie">Bezorgfrequentie:
+                <br />
+                <ul>
+                    <li>
+                <input {...register("Bezorgfrequentie", {required: true})} type="radio"
+                       value="iedere week"/>iedere week</li>
+                    <li>
+                <input {...register("Bezorgfrequentie", {required: true})} type="radio"
+                       value="om de week "/>om de week</li>
+                    <li>
+                <input {...register("Bezorgfrequentie", {required: true})} type="radio"
+                       value="iedere maand"/>iedere maand</li>
+                    <li>
+                <input {...register("Bezorgfrequentie", {required: true})} type="radio"
+                       value="anders"/>anders</li>
+
+                </ul>
+            </label>
+            {selectedReferrer === 'anders' && (
+                <input
+                    type="text"
+                    {...register("Bezorgfrequentie-anders", {required: true})}/>
+            )}
+
+            <br />
+
+            <label htmlFor="comments">Opmerkingen:
+                <br />
+                <textarea name="comments" id="comments" cols="30" rows="10"
+                          {...register("comments")}
+                >
+                            </textarea>
+            </label>
+            <label htmlFor="terms-and-conditions">
+                <input
+                    type="checkbox"
+                    id="terms"
+                    name="terms"
+                    checked={checkedTerms}
+                    onChange={() => toggleCheckedTerms(!checkedTerms)}
```

Comment:
In principe is dit geen 'toggle', omdat je de waarde moet meegeven. Maar dat is puur een naamdingetje natuurlijk